doc/api/crypto.md

```diff
@@ -1736,7 +1736,7 @@ changes:
 -->
 
 * `algorithm` {string}
-* `key` {string | Buffer | TypedArray | DataView}
+* `key` {string | Buffer | TypedArray | DataView | KeyObject}
 * `iv` {string | Buffer | TypedArray | DataView | null}
```

Comment:
Note to reviwer) `| null` was added in https://github.com/nodejs/node/pull/29684/files#diff-29db77451cd94ef9fbab42fd5b7499d5R1740